hcipy/aperture/realistic.py

```diff
@@ -106,7 +107,7 @@ def make_subaru_aperture():
 def make_lbt_aperture():
 	pass
 
-def make_magellan_aperture(normalized=False, with_spiders=True):
+def make_magellan_aperture(normalized=False, with_spiders=True, oversize=1.0):
```

Comment:
I did not see this the first time. `oversize`?

Reply:
That has to be removed. Done